salt/channel/server.py

```diff
@@ -375,7 +371,7 @@ def _auth(self, load, sign_messages=False):
         elif os.path.isfile(pubfn):
             # The key has been accepted, check it
             with salt.utils.files.fopen(pubfn, "r") as pubfn_handle:
-                if self._clean_key(pubfn_handle.read()) != self._clean_key(load["pub"]):
+                if salt.crypt.clean_key(pubfn_handle.read()) != load["pub"]:
```

Comment:
I'm wondering why the right side lose it's `clean_key` ?

This break minions authentication that have been accepted in the past with spaces at the end of their public keys.